app/entrypoint.sh

```diff
@@ -2,17 +2,19 @@
 
 # Run npm build
 export PATH=$PATH:/home/django/.nvm/versions/node/v12.18.3/bin
+cd /react-client-app
 npm run build
 
 # Wait until DB is available
 wait-for-it ${COCONNECT_DB_HOST}:${COCONNECT_DB_PORT} -- echo "Database is ready! Listening on ${COCONNECT_DB_HOST}:${COCONNECT_DB_PORT}"
 
-# Collect static files for serving
 cd /api
+
+# Collect static files for serving
 rm -rf staticfiles
 mkdir staticfiles
-python /api/manage.py collectstatic
-python /api/manage.py migrate
+python manage.py collectstatic
```

Comment:
I think this all needs testing - I am hesitant to change anything in `entrypoint.sh` given it is used by the deployment as well. 
Ideally the only changes would be in `Dockerfile`, not `entrypoint`

Reply:
Yeah, I see your point.
I left the entrypoint untouched and tested everything again. Everything worked fine on my end. However, it would be really helpful if you could spare some time to test the docker-compose on your end as well. Thanks

Reply:
Perfect - thanks for that, this should work with the existing deployment then. It runs from scratch pulling it down for me, great job,